packages/cli/src/Format.ts

```diff
@@ -56,10 +56,15 @@ Or specify a Prisma schema path
 
     const output = await formatSchema({ schemaPath })
 
-    // Validate whether the formatted output is a valid schema
-    await getDMMF({
-      datamodel: output,
-    })
+    try {
+      // Validate whether the formatted output is a valid schema
+      await getDMMF({
+        datamodel: output,
+      })
+    } catch (e) {
+      console.error('') // empty line for better readability
+      throw new Error(`${e.message}`)
+    }
```

Comment:
Ok for now, but in the future I hope we'll use something other than `getDMMF` for the purpose of ensuring the absence of errors

Reply:
We only have that for now